zgw/zaken-api/src/main/kotlin/com/ritense/zakenapi/event/ZaakRolUpdated.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2015-2024 Ritense BV, the Netherlands.
```

Comment:
I think your configured copyright might be out of date, it should be 2025 instead of 2024.

```suggestion
 * Copyright 2015-2025 Ritense BV, the Netherlands.
```